app/src/main/java/id/rizmaulana/covid19/util/ext/ViewExt.kt

```diff
@@ -43,4 +46,17 @@ fun View.gone() {
 
 fun Context.color(resource: Int): Int {
     return ContextCompat.getColor(this, resource)
-}
\ No newline at end of file
+}
+
+fun RecyclerView.ViewHolder.getString(@StringRes string: Int): String {
+   return itemView.context.getString(string)
+}
+
+
+fun  RecyclerView.ViewHolder.getStringWithArg(@StringRes string: Int, arg: String): String {
+    return itemView.context.getString(string, arg)
+}
```

Comment:
@rizmaulana we could use `vararg` in the second param,  so we can add multiple arguments if the string res need it.

Reply:
Already try use varargs, but instead extracting string into string resource, it printing the object name like "LObjec()...", have an ideas mas @pandubaraja ?

Reply:
I think you forgot to use the spread operator, like this @rizmaulana 

```
fun  RecyclerView.ViewHolder.getStringWithArg(@StringRes string: Int, vararg args: Any): String {
    return itemView.context.getString(string, *args)
}
```

This is working I've tried it